doc/Language/101-basics.rakudoc

```diff
@@ -461,6 +461,34 @@ say "we have @flavors.sort.join(', ')";
                                   # OUTPUT: «we have peach, vanilla␤»
 =end code
 
+=head1 Calling a raku method from another file
+
+To call a raku subroutine from another file, the convention is to have a minimal
+raku module or class file.
```

Comment:
To address @lizmat's concern, maybe we can note this is intended to be a simplistic example and point to how to write a module for a more... realistic? larger? example?